File: BackEnd/src/main/java/com/omnifood/omnifoodorder/model/Order.java
package com.omnifood.omnifoodorder.model;

import com.fasterxml.jackson.annotation.JsonIgnore;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name = "order")
public class Order extends CategoryOrder {

    @Column
    private int price;
    @Column
    private String image;
    @Column
//    @Lob
    private String description;

    @JsonIgnore
    @ManyToOne
    @JoinColumn(name = "categoryId")
    private Category category;

}

File: FrontEnd/src/app/common/model/order.ts

export class Order {
    id: number;
    name: string;
    dataCreate: Date;
    dataUpdate: Date;
    description: string;
    image: string;
    price: number;


    constructor(id: number, name: string, dataCreate: Date, dataUpdate: Date, description: string, image: string, price: number) {
        this.id = id;
        this.name = name;
        this.dataCreate = dataCreate;
        this.dataUpdate = dataUpdate;
        this.description = description;
        this.image = image;
        this.price = price;
    }
}
